packages/fuels-signers/src/wallet.rs

```diff
@@ -222,6 +225,46 @@ impl Wallet {
             .map_err(Into::into)
     }
 
+    pub async fn get_messages(&self) -> Result<Vec<schema::message::Message>, Error> {
+        Ok(self.get_provider()?.get_messages(&self.address).await?)
+    }
+
+    pub async fn get_inputs_for_messages(&self, witness_index: u8) -> Result<Vec<Input>, Error> {
+        let as_u8_slice = |v: &[i32]| -> &[u8] {
```

Comment:
Albeit it does look safe I'd rather we use safe code that can achieve the same effect, unless the performance impact is measurable:
```rust
        let to_u8_bytes = |v: &[i32]| { v.iter().flat_map(|e| e.to_be_bytes()).collect::<Vec<_>>() };
```
You'll have a few allocations but you'll sleep _safe_

Reply:
Although this probably needs some thinking through...

If you're on a little endian platform (which you probably are) and have some numbers also in LE and want to have the underlying bytes then you'd call `to_le_bytes` 

Thankfully there is `to_ne_bytes` for native endianess.